R/my_first_fcn.R

```diff
@@ -0,0 +1,14 @@
+#' Welcome message
+#' @param hw TRUE or FALSE
```

Comment:
We expect a little more description here in general about what the parameter does